src/transformers/modeling_flax_utils.py

```diff
@@ -820,31 +891,35 @@ def from_pretrained(
                 _commit_hash=commit_hash,
             )
 
+        safetensors_from_pt = False
+        if filename == SAFE_WEIGHTS_NAME:
+            with safe_open(resolved_archive_file, framework="flax") as f:
+                safetensors_metadata = f.metadata()
+            if safetensors_metadata is None or safetensors_metadata.get("format") not in ["pt", "tf", "flax"]:
+                raise OSError(
+                    f"The safetensors archive passed at {resolved_archive_file} does not contain the valid metadata."
+                    " Make sure you save your model with the `save_pretrained` method."
+                )
+            safetensors_from_pt = safetensors_metadata.get("format") == "pt"
+            safetensors_metadata.get("format") == "tf"
```

Comment:
Is this line here by accident?

Reply:
Ah yes, typo from trying to enable loading from TensorFlow safetensors :grin: Thanks for the catch!